mautic/mautic
# fix:[DPMMA-2213] fix chart focus view statistics

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ Yes]
| New feature/enhancement? (use the a.x branch)      | [ No]
| Deprecations?                          | [ No]
| BC breaks? (use the c.x branch)        | [No ]
| Automated tests included?              | [Yes] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Fix issue: #13583
Incorrect Statistics Counting in Focus Items Chart 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))

2. Display the Landing page with Focus Item.
3. On the views chart: 1 (or increased by 1).
4. Click on a link in the focus item.
5. On the chart, both clicks and views are increased by 1 (clicks: 1, views: 2).
6. Click on the remaining 2 links. On the chart, clicks: 3, views: 4.

Expected Outcome:
Clicking on a link in the focus item should not increase the number of views.
